website/docs/r/policy_set_definition.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_policy_set_definition"
+sidebar_current: "docs-azurerm-resource-policy-set-definition"
+description: |-
+  Manages a policy set definition.
+---
+
+# azurerm_policy_set_definition
+
+Manages a policy set definition. 
+
+-> **NOTE:**  Policy set definitions (also known as policy initiatives) do not take effect until they are assigned to a scope using a Policy Set Assignment.
+
+## Example Usage
+
+```hcl
+resource "azurerm_policy_set_definition" "test" {
+  name         = "testPolicySet"
+  policy_type  = "Custom"
+  display_name = "Test Policy Set"
+
+  parameters = <<PARAMETERS
+    {
+        "allowedLocations": {
+            "type": "Array",
+            "metadata": {
+                "description": "The list of allowed locations for resources.",
+                "displayName": "Allowed locations",
+                "strongType": "location"
+            }
+        }
+    }
+PARAMETERS
+
+  policy_definitions = <<POLICY_DEFINITIONS
+    [
+        {
+            "parameters": {
+                "listOfAllowedLocations": {
+                    "value": "[parameters('allowedLocations')]"
+                }
+            },
+            "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/e765b5de-1225-4ba3-bd56-1ac6695af988"
+        }
+    ]
+POLICY_DEFINITIONS
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the policy set definition. Changing this forces a new resource to be created.
+
+* `policy_type` - (Required) The policy set type. The value can be `BuiltIn`, `Custom` or `NotSpecified`
```

Comment:
We tend to phrase this as:

```suggestion
* `policy_type` - (Required) The policy set type. Possible values are `BuiltIn` or `Custom`. Changing this forces a new resource to be created.
```

(note I've removed `NotSpecified` since I don't believe we need to support it)